docs/release-notes/cloud/cloud-2024-04.md

```diff
@@ -0,0 +1,100 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud April 2024.
+included_jira_versions: ['2024.04']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.272
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.305
+---
+
+# Cloud April 2024
+
+These release notes are for the Codacy Cloud updates during April 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues with disabled release notes
+
+Bugs and other issues:
+-   https://codacy.atlassian.net/browse/ALA-1000
+-->
+
+## Product enhancements
+
+-   We revamped the [Security and risk management](../../organizations/managing-security-and-risk.md) area. (TAROT-2546)
+
+    ![Security and risk management](../images/tarot-2546.png)
+
+-   We revamped the **Commit** and **Pull Request** detail pages. (ALA-639)
```

Comment:
@nicklem Feel free to tweak this one also. 🤓

Reply:
Given it's a migration and the overall capabilities of the pages are comparable, I'm OK with just mentioning the associated feature removals. Thoughts?

Reply:
Fair enough. I was assuming there were relevant changes to the overall capabilities of the pages, but if that's not the case we can stick with the mention on the feature removals.

Reply:
```suggestion
```